core-services/prow/02_config/maistra/_prowconfig.yaml

```diff
@@ -1,6 +1,7 @@
 branch-protection:
   orgs:
     maistra:
+      protect: true
       unmanaged: true
```

Comment:
I have no idea what those settings mean. Can they coexist?

Reply:
well, I can remove the "protect:" one. The branch-protection has been in maistra repos. Keep "unmanaged" can avoid overwriting the existing github branch config.

Reply:
sounds good.